generators/client/templates/angular/src/main/webapp/app/account/password/password.component.html.ejs

```diff
@@ -38,7 +38,8 @@
                     <label class="form-control-label" for="currentPassword" jhiTranslate="global.form.currentpassword.label">Current password</label>
                     <input type="password" class="form-control" id="currentPassword" name="currentPassword"
                            placeholder="{{ 'global.form.currentpassword.placeholder' | translate }}"
-                           formControlName="currentPassword">
+                           formControlName="currentPassword"
+                           data-cy="currentPassword">
```

Comment:
Here the same name is used. For consistency, we should pick a "rule"

Reply:
I think this is a mistake and we should put the same name than formControlName is the data-cy. easier